components/TeamMember/TeamMember.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import Image from "next/image";
+import { Link } from "../Link/Link";
+import { ContactLink, Img } from "types/layout-schema-types";
+import { urlFor } from "lib/client";
+
+interface TeamMemberProps extends Partial<React.HTMLProps<HTMLElement>> {
```

Comment:
I'm curious about this, why do you need the `Partial<T>`?

Reply:
> I'm curious about this, why do you need the `Partial<T>`?

You're right, it's not necessary